app-creator/client/src/widgets/app-root.ts

```diff
@@ -223,6 +262,34 @@ export class AppRoot extends LitElement {
       ? html`<paper-progress indeterminate></paper-progress>`
       : nothing;
 
+    const sortingChips = html`
+      <div id="chips-container">
+        <paper-chip
+          selectable
+          class="${this.deviceFilter === DeviceType.all
+            ? 'selected-paper-chip'
+            : ''}"
+          @click=${() => handleDeviceFilters.call(this, DeviceType.all)}
```

Comment:
Since we are passing the device type as an argument, I needed to wrap it with an anonymous function. The context `this` is only available in the anonymous function and not in the handleDeviceFilters function and thus I needed to bind it. Alternatively I could do this:

```Typescript
() => {
  this.deviceFilter = device;
}
```